modules/ROOT/pages/clustering/seed.adoc

```diff
@@ -69,6 +69,12 @@ neo4j-03$ ./bin/neo4j start
 +
 The cluster forms and the replicated Neo4j DBMS deployment comes online.
 
+[NOTE]
+====
+The system database defines the databases that should exist in your neo4j instance.
+If a database doesn't exist in your system database (because it hasn't been created previously) it will have to be created with `CREATE DATABASE <database-name>` even if it has been restored/loaded/imported.
```

Comment:
```suggestion
If a database does not exist in your system database (because it has not been created previously), it must be created with `CREATE DATABASE <database-name>` even if it has been restored/loaded/imported.
```